src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -112,7 +112,7 @@ def _get_feature_in_preview_message():
 register_cli_argument('acs create', 'windows', action='store_true', help='If true, set the default osType of agent pools to be Windows.')
 register_cli_argument('acs create', 'validate', action='store_true', help='Generate and validate the ARM template without creating any resources')
 
-register_cli_argument('acs create', 'orchestrator_release', options_list=('--orchestrator-release',), help=_get_feature_in_preview_message() + 'Use Orchestrator Release to specify the major.minor part of the semantic version for your choice of orchestrator. For example, you can specify 1.7, which will resolve to 1.7.x based on ACS latest well tested batch version')
+register_cli_argument('acs create', 'orchestrator_version', options_list=('--orchestrator-version',), help=_get_feature_in_preview_message() + 'Use Orchestrator Version to specify the semantic version for your choice of orchestrator.')
 
```

Comment:
Isn't this a breaking change?

Reply:
It is a breaking change.

We have the breaking change in swagger as well. It is something we have to do, and fully discussed.
Btw, it is only for 2017-07-01 api version, which is in public preview, which means we could do breaking changes.

Reply:
This would also be seen by customers who are still using the old api-version and not using the preview.
Is this intended? or should a check for the api_version be made here before the parameter is registered?

Reply:
It is intended. Every parameter only available for preview is clearly stated int the description. It is good for now. We don't want to hide parameters.

Reply:
Cool, makes sense.